var/spack/repos/builtin/packages/faodel/package.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Faodel(CMakePackage):
+    """Flexible, Asynchronous, Object Data-Exchange Libraries"""
+
+    homepage = "https://github.com/faodel/faodel"
+    url      = "https://github.com/faodel/faodel/archive/v1.1811.1.tar.gz"
+    git      = "https://github.com/faodel/faodel.git"
+
+    maintainers = ['fbudin69500', 'chuckatkins']
```

Comment:
Please add yourself as the maintainer, instead of the two of us.  We're working on improving the packaging for many of the data and viz products as part of the SDK, but ultimately the project itself owns it

Reply:
Will do.